src/test/javascript/spec/component/course/course-scores/course-scores.component.spec.ts

```diff
@@ -374,6 +375,7 @@ describe('CourseScoresComponent', () => {
 
         expect(component.allParticipationsOfCourse).toEqual(participations);
         expect(component.maxNumberOfOverallPoints).toEqual(overallPoints);
+        // @ts-ignore
```

Comment:
`// @ts-ignore` should be avoided and we should consider making `exerciseMaxPointsPerType` value public

Reply:
Sure, I’m not a fan of making the attribute public just for the test but I removed it.